### ISSUE TITLE
For AISW-52469: Keras BatchNorm Folding Failure for Unet Model

### ISSUE THREAD

**Issue by quic-ristha** [quic/aimet#1693]
Batchnorm folding  into Conv2DTranspose failing.
